objects/BlessCurseManager.5933fb.json

```diff
@@ -33,8 +33,8 @@
   "IgnoreFoW": false,
   "LayoutGroupSortIndex": 0,
   "Locked": true,
-  "LuaScriptState": "false",
   "LuaScript": "require(\"tokens/BlessCurseManager\")",
+  "LuaScriptState": "false",
```

Comment:
This shouldn't matter, but it would be safer to just remove objects from the PR that aren't supposed to be edited

Reply:
I believe this is now resolved?